### ISSUE TITLE
How can we generate bilinear interpolation wms layer using this or sld 

### ISSUE THREAD

**Issue by chakravarthi-vl** [Reading-eScience-Centre/ncwms#49]
****

**Comment by guygriffiths** [comment]
You can't.  ncWMS reads data in a nearest-neighbour fashion onto an image grid.  By the time any SLDs are applied, the data has already been quantised, with no chance of interpolation.

**Comment by chakravarthi-vl** [comment]
Ok, is there any way we can get smooth curves rather than grid box in wms layer?

**Comment by guygriffiths** [comment]
No, ncWMS is really designed to quickly and accurately display data.  If data appears blocky, it's because the data _is_ blocky.  The only way to get smooth data is to use underlying data which is of a high enough resolution.

**Comment by chakravarthi-vl** [comment]
Ok Thanks, May I know what is the smallest possible grid size to convert blocks to curves.
Is there any WMS server that you know which will give smooth curves from grid data.

**Comment by guygriffiths** [comment]
I'm not sure I know what you mean.  If you are plotting no more than 1 pixel per grid data point, an image will appear smooth.  If each grid data point corresponds to multiple pixels, it will look blocky.  The numbers will depend entirely on the underlying resolution and the area being visualised.

I think Geoserver has some interpolation abilities, but it can't directly plot NetCDF as far as I'm aware.

**Comment by chakravarthi-vl** [comment]
Hi Thanks for Reply,
I want to develop like https://www.windy.com  from NetCDF file's, If you any procedure or server let me know

**Comment by guygriffiths** [comment]
What's your purpose for such an application?  I'm currently working for https://www.the-iea.org/ and continuing ncWMS support on the side.  If your organisation has any funding available we might be able to produce something like that (at least a working prototype), but I don't have enough free time to develop that in my own time.
